lib/job_queue/forward_model.cpp

```diff
@@ -208,6 +209,11 @@ static void forward_model_json_fprintf(const forward_model_type * forward_model,
   fprintf(stream, "}\n");
   fclose(stream);
   free(json_file);
+
+  char * status_file = (char*)util_alloc_filename(path , DEFAULT_STATUS_JSON, NULL);
```

Comment:
I find it a bit awkward that there is removing of a file within an fprintf function, but I understand the predicament of having the cleaning placed in the same location as the creation (since the file isn't created before the first job has been run..) - noted in https://github.com/Statoil/libres/issues/478